static/js/zamboni/buttons.js

```diff
@@ -274,6 +274,7 @@ var installButton = function() {
             warn(gettext('Not available for your platform'));
             $button.addClass('concealed');
             $button.first().css('display', 'inherit');
+            $button.closest('.item.addon').addClass('incompatible');
```

Comment:
I'm assuming we modified some CSS that prevented it doing something directly to the CSS in JS. But I'm honestly not clear on that.